generate_function_reference.py

```diff
@@ -29,6 +29,12 @@
         any_value(tags) AS func_tags,
     FROM duckdb_functions() as funcs
     WHERE function_type = '$FUNCTION_TYPE$'
+        -- function-specific tweaks
+        AND CASE function_name
+            -- TODO: https://github.com/duckdb/duckdb-spatial/pull/601#discussion_r2144753435
+            WHEN '&&' THEN 'box' IN parameters
+            ELSE true
+        END
```

Comment:
This isn't nice, but I think `&&`'s document should really note that the operation is not very accurate.